exporter/metrics.go

```diff
@@ -265,24 +266,44 @@ func makeMetrics(prefix string, m bson.M, labels map[string]string, compatibleMo
 				continue
 			}
 
-			metrics := []*rawMetric{rm}
+			// metrics := []*rawMetric{rm}
```

Comment:
these changes were done to be able to keep `mongodb_mongod_instance_uptime_seconds` and `mongodb_instance_uptime_seconds` together.